lib/Mojo/Asset/File.pm

```diff
@@ -49,7 +49,7 @@ sub contains {
   $handle->sysseek($self->start_range, SEEK_SET);
 
   # Calculate window size
-  my $end  = $self->end_range // $self->size;
+  my $end  = $self->_end_range // $self->size;
```

Comment:
Could _end_range() return $self->size instead of undef?

Reply:
Good idea and makes the golf easier